### ISSUE TITLE
No Data found after PlotPredictions method has been run

### ISSUE THREAD

**Issue by orionnelson** [TimRivoli/Stock-Price-Trade-Analyzer#4]
Test accuracy: 0.671875 loss: nan
Running predictions...
Predictions complete.

Warning (from warnings module):
  File "C:\Users\User\AppData\Local\Programs\Python\Python36\lib\site-packages\pandas\core\frame.py", line 4259
    **kwargs
SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
 Saved to data/charts/GME_30days.png
Statistics saved to: data/historical/GME_stats_predictions.csv
Plotting predictions for BA
BA

[*********************100%***********************]  1 of 1 completed
 No data found for BA

**Comment by orionnelson** [comment]
After PlotPredictions is run it seems that data cannot be loaded. I was also looking into the plot predictions method could not find what was causing the issue. 
There is also another weird issue where GraphData method reports weird dates when predicting a small number of days.

If there is any way you can share an requirments.txt with versions of packages you have installed I feel that I have got most of them in the required versions but am still having some issues.

**Comment by TimRivoli** [comment]
It looks like you are running predictions for two tickers and on the second one data is failing to load which then produces random data in the graph of predicted days.  I've seen this happen when the second ticker fails training but it works when you run it by itself.  Try running just BA and see if it works.  It might be some type of memory re-allocation problem in one of the Python libraries.

**Comment by orionnelson** [comment]
Yes the issue just described is exactly what Im experiencing.  Have you found any way around it.  I have tried running each individual ticker in a new shell but the issue still persists.
